source/manage/logging.rst

```diff
@@ -0,0 +1,487 @@
++---------------------+----------+-----------------------------------------------------------------------------------------+
+| timestamp_format    | string   | Format for timestamps. Default is `RFC3339 <https://www.rfc-editor.org/rfc/rfc3339>`__. |
++---------------------+----------+-----------------------------------------------------------------------------------------+
+
+GELF log format configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++----------+----------+----------------------------------------------------------+
+| **Key**  | **Type** | **Description**                                          |
++----------+----------+----------------------------------------------------------+
+| hostname | string   | Outputs a custom hostname in log records.                |
+|          |          | If omitted, hostname is taken from the operating system. |
++----------+----------+----------------------------------------------------------+
+
+Configure log levels and events
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++------------+----------+--------------------------------------------------------------+
+| **Key**    | **Type** | **Description**                                              |
++------------+----------+--------------------------------------------------------------+
+| id         | number   | Unique identifier of the log level.                          |
++------------+----------+--------------------------------------------------------------+
+| name       | string   | Name of the log level.                                       |
++------------+----------+--------------------------------------------------------------+
+| stacktrace | bool     | Outputs a stack trace. Default is ``false``.                 |
++------------+----------+--------------------------------------------------------------+
+| color      | number   | The ANSI color code used to output parts of the log record.  |
+|            |          | Supported values include:                                    |
+|            |          |                                                              |
+|            |          | - Black: ``30``                                              |
+|            |          | - Red: ``31``                                                |
+|            |          | - Green: ``32``                                              |
+|            |          | - Yellow: ``33``                                             |
+|            |          | - Blue: ``34``                                               |
+|            |          | - Magenta: ``35``                                            |
+|            |          | - Cyan: ``36``                                               |
+|            |          | - White: ``37``                                              |
++------------+----------+--------------------------------------------------------------+
+
+Log levels
```

Comment:
Should we also include all the log levels for standard logs in this table?

```
 {"ID": 5, "Name": "debug", "Stacktrace": false},
            {"ID": 4, "Name": "info", "Stacktrace": false},
            {"ID": 3, "Name": "warn", "Stacktrace": false},
            {"ID": 2, "Name": "error", "Stacktrace": true},
            {"ID": 1, "Name": "fatal", "Stacktrace": true},
            {"ID": 0, "Name": "panic", "Stacktrace": true}
```

Reply:
@wiggin77 - Thoughts?